portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,33 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      [ 'I was born in Venezuela, so I am a native Spanish speaker', 
+      'I learned to play the violin when I was 7 years old', 
```

Comment:
Do you? I learned to play violin when I was 9, although I hate practice at that time.